modules/tinymce/src/core/main/ts/api/util/Observable.ts

```diff
@@ -78,7 +89,7 @@ const Observable: Observable<any> = {
       let parent = self.parent();
       while (parent && !dispatcherArgs.isPropagationStopped()) {
         parent.dispatch(name, dispatcherArgs, false);
-        parent = parent.parent();
+        parent = parent.parent ? parent.parent() : undefined;
```

Comment:
Code change here, as while everything that uses this doesn't implement `parent()`, if it did this may have thrown an exception.